frontend/src/pages/devices/Detail.tsx

```diff
@@ -50,10 +50,10 @@ const Detail = () => {
     setRevision(revision + 1)
   }
 
-  const m = useRouteMatch("/" + Endpoint + "/:id")
```

Comment:
Using the `Endpoint` variable can help with making sure we're consistent if we ever change our route (or more likely, copy/paste this code for a new detail page).